index.js

```diff
@@ -533,6 +535,10 @@ function Argv (processArgs, cwd) {
     })
   }
 
+  function guessLocale () {
+    return osLocale.sync().split('_')[0]
```

Comment:
@bcoe FYI `y18n@3.1.0` has been released and should remove the need to split result of `osLocale.sync()`